### ISSUE TITLE
better topic creation

### ISSUE THREAD

**Issue by Aokromes** [telegramdesktop/tdesktop#28285]
### Is your feature request related to a problem?

atm creating topics is slow having to do multiple steps on multiple places

### Describe the solution you'd like

allow to write on create topic window the pinned message
checkbox to pin topic.
checkbox to close topic.
mute topic dropdown.

### Describe alternatives you've considered

related:
https://github.com/telegramdesktop/tdesktop/issues/27271

**Comment by 23rd** [comment]
Did you see something similar in the other official clients?

**Comment by Aokromes** [comment]
nope, but atm to much steps are needed to  create a new pinned locked topic :)

**Comment by buttler2** [comment]
Good idea
